clock/app/helpers/stores_helper.rb

```diff
@@ -1,2 +1,9 @@
 module StoresHelper
+  def message_helper
```

Comment:
I can't see this helper method in the view, are we going to use it later? maybe in another view? if the answer is no, I think we could remove it

Reply:
We used the Rails scaffold, because of that the helper was also generated. Although the idea was also to use it in the logging, but better we will use Devise.